test/src/test/java/jenkins/util/JenkinsJVMRealTest.java

```diff
@@ -17,7 +17,7 @@
     public static JenkinsRule j = new JenkinsRule();
 
     @Test
-    public void isJenkinsJVM() throws Exception {
+    public void isJenkinsJVM() throws Throwable {
```

Comment:
The build logs exhibiting the issue: https://ci.jenkins.io/job/Core/job/jenkins/job/PR-2802/2/execution/node/28/log/

Copying the end for future ref when the build has been wiped out:

```
proprietary API and may be removed in a future release
21:18:06 [INFO] 100 warnings 
21:18:06 [INFO] -------------------------------------------------------------
21:18:06 [INFO] -------------------------------------------------------------
21:18:06 [ERROR] COMPILATION ERROR : 
21:18:06 [INFO] -------------------------------------------------------------
21:18:06 [ERROR] /home/jenkins/workspace/Core_jenkins_PR-2802-7MEOUWG2NJTLSSLTVDVDIXOKU5BJ5UJ5PCL2QYQPQXAHRMFCDSMA/test/src/test/java/jenkins/util/JenkinsJVMRealTest.java:[23,42] error: unreported exception T; must be caught or declared to be thrown
21:18:06 [ERROR] /home/jenkins/workspace/Core_jenkins_PR-2802-7MEOUWG2NJTLSSLTVDVDIXOKU5BJ5UJ5PCL2QYQPQXAHRMFCDSMA/test/src/test/java/hudson/model/GetEnvironmentOutsideBuildTest.java:[89,30] error: no suitable method found for buildAndAssertSuccess(AbstractProject)
21:18:06 [INFO] 2 errors 
21:18:06 [INFO] -------------------------------------------------------------
21:18:06 [INFO] ------------------------------------------------------------------------
21:18:06 [INFO] Reactor Summary:
21:18:06 [INFO] 
21:18:06 [INFO] Jenkins main module ................................ SUCCESS [ 45.970 s]
21:18:06 [INFO] Jenkins cli ........................................ SUCCESS [ 42.146 s]
21:18:06 [INFO] Jenkins core ....................................... SUCCESS [03:12 min]
21:18:06 [INFO] Jenkins war ........................................ SUCCESS [01:23 min]
21:18:06 [INFO] Tests for Jenkins core ............................. FAILURE [ 56.812 s]
21:18:06 [INFO] ------------------------------------------------------------------------
21:18:06 [INFO] BUILD FAILURE
21:18:06 [INFO] ------------------------------------------------------------------------
21:18:06 [INFO] Total time: 07:18 min
21:18:06 [INFO] Finished at: 2017-03-29T21:18:05+00:00
21:18:06 [INFO] Final Memory: 100M/594M
21:18:06 [INFO] ------------------------------------------------------------------------
21:18:06 [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.0:testCompile (default-testCompile) on project test: Compilation failure: Compilation failure:
21:18:06 [ERROR] /home/jenkins/workspace/Core_jenkins_PR-2802-7MEOUWG2NJTLSSLTVDVDIXOKU5BJ5UJ5PCL2QYQPQXAHRMFCDSMA/test/src/test/java/jenkins/util/JenkinsJVMRealTest.java:[23,42] error: unreported exception T; must be caught or declared to be thrown
21:18:06 [ERROR] /home/jenkins/workspace/Core_jenkins_PR-2802-7MEOUWG2NJTLSSLTVDVDIXOKU5BJ5UJ5PCL2QYQPQXAHRMFCDSMA/test/src/test/java/hudson/model/GetEnvironmentOutsideBuildTest.java:[89,30] error: no suitable method found for buildAndAssertSuccess(AbstractProject)
21:18:06 [ERROR] -> [Help 1]
21:18:06 [ERROR] 
21:18:06 [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
21:18:06 [ERROR] Re-run Maven using the -X switch to enable full debug logging.
21:18:06 [ERROR] 
21:18:06 [ERROR] For more information about the errors and possible solutions, please read the following articles:
21:18:06 [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
21:18:06 [ERROR] 
21:18:06 [ERROR] After correcting the problems, you can resume the build with the command
21:18:06 [ERROR]   mvn <goals> -rf :test
```